src/System.Net.Security/src/System/Net/Windows/CertModule.cs

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Windows code have not touched upon mostly except where fitting was required to make SSPIWrapper & above layer agnostic of underlying os types.